apps/air-discount-scheme/backend/src/app/modules/flight/flight.service.ts

```diff
@@ -27,6 +27,9 @@ export const ADS_POSTAL_CODES = {
   Vestmannaeyjar: 900,
 }
 const DEFAULT_AVAILABLE_LEGS = 6
+
+// This doesn't need to be updated year by year
+// unless deviations from the 6 default flightlegs occur
```

Comment:
Line 186 checks whether the current year is listed. 
If it is listed, overwrite `availableLegsThisYear`.
Line 185 sets this variable `availableLegsThisYear`.

Am I missing something because I don't see how it will fail.

The includes function simply returns false in the event that the year is not listed, skipping the if statement body.